### ISSUE TITLE
Keeping original CreationTime during synchronization

### ISSUE THREAD

**Issue by tgrabows** [aegif/CmisSync#755]
I have problem with CmisSync 2.11.3.0 on Windows Server 2012R2 and Alfresco Community 5.2 on Debian 9.

I set up synchronization between CmisSync and Alfresco. Generally, I see no problems, but I have one issue. When I insert new file for synchronization, and it is copied to ECM, it lost original CreationTime.  Alfresco Created Date is fulfilled by current time.

Best regards,
Tomasz Grabowski

**Comment by nicolas-raoul** [comment]
Could you please post screenshots showing the times?
Thanks for the feedback!

**Comment by tgrabows** [comment]
I have already inserted file `old-file.html`. Its `CreationTime` is `2017-06-15 00:02`.

![windows](https://user-images.githubusercontent.com/33065613/31987889-90073ae6-b96d-11e7-988f-21bd398c8ac4.png).

In Alfresco repository I see its `Created Date: 25 Oct 2017 09:21:20`.

![alfresco](https://user-images.githubusercontent.com/33065613/31988082-2e065826-b96e-11e7-9878-817c492f5b03.png)

I would like to have original `CreationTime: 2017-06-15 00:02` :)

**Comment by douglascrp** [comment]
I am afraid that wont happen even if you create upload the fine into Alfresco using the Share interface.
That date is controlled by the audit, and it the exact moment the object was created inside Alfresco, and it has nothing to do with the original date when the document was created in your computer.

In order to change that behaviour, you would have to write some Java code, and disable the audit policy, but this is totally out of context for this project.

**Comment by tgrabows** [comment]
You're right, this is also not happening while uploading files through Share.
This is understandable, because POST request from web browser doesn't contain file system metadata (ex. CreationTime, ModificationTime, permission, etc.).

I expected, that it is another layer problem.

**Comment by nicolas-raoul** [comment]
Indeed, some users probably expect to see creation date as the date at which the document entered Alfresco.
But some other users like @tgrabows expect the creation date to be the creation date on the local filesystem.
I think both ways of thinking are right (everyone feel free to tell us which one you prefer), so that might be an option we could want in the future. Don't expect it any time soon, though.
Thanks for the feedback!
